src/discord_utils.py

```diff
@@ -31,6 +31,54 @@ async def try_set_pin(message: Message, pin_state: bool) -> None:
         print("Altering message pin state failed:", e)
 
 
+def build_filepath_for_attachment(message_id: int, attachment: Attachment) -> str:
+    """
+    Generate a unique filepath for a given attachment for the purposes of saving it to disk
+
+    Generates a filepath in the following format:
```

Comment:
imo all these lines specifying the exact format are unnecessary to know how to use the function and just take up comment space, especially considering the code right below which actually does this is fairly readable. The comment should be for stuff that's not so understandable directly from code below.

Reply:
This comment still applies lol

Reply:
I see it doesn't necessarily show the previous comment in the timeline so I'll repaste it

> imo all these lines specifying the exact format are unnecessary to know how to use the function and just take up comment space, especially considering the code right below which actually does this is fairly readable. The comment should be for stuff that's not so understandable directly from code below.